index.html

```diff
@@ -133,13 +156,21 @@ <h4>My Project Portfolio</h4>
     
 </div>
 </section>
+</div>
+<div class="bottom-about">
+<div class="About-bottom">
+    <img src="asset/Groupshape.png" alt="">
+</div>
+</div>
```

Comment:
@cwaku i cant understand this statement can you explain more

Reply:
I don't have much time again. this is why I use this method @cwaku

Reply:
It means that these sections should be made using HTML and CSS instead of images: :grin: 

- [About section](https://github.com/cynthia-nwannah/my-Project-portfolio/blob/f10b2aeefb2ca6449cbd2532b60fdc5d800854aa/asset/Groupshape.png)
- [Form section](https://github.com/cynthia-nwannah/my-Project-portfolio/blob/f10b2aeefb2ca6449cbd2532b60fdc5d800854aa/asset/Groupshape2.png)
- [Works section](
https://user-images.githubusercontent.com/78335447/171773365-a1897a7e-3b8a-40f3-b495-dd2552f5de2a.png)